x-pack/libbeat/cmd/inject.go

```diff
@@ -7,8 +7,9 @@ package cmd
 import (
 	"github.com/elastic/beats/libbeat/cmd"
 
-	// register central management
+	// register central management modules
```

Comment:
Should we really call this also modules? All things modules? :-D

Reply:
it should say packages, I wonder what I was thinking of :man_facepalming: :) will push a change